src/fstests/fstests.jinja2

```diff
@@ -5,9 +5,17 @@
 
 {%- block python_imports %}
 {{ super() }}
+import json
 import subprocess
+import xml.etree.ElementTree as ET
 {%- endblock %}
 
+{%- block python_globals %}
+{{ super() }}
+FSTESTS_ROOT = '/home/kernelci/alex/fstests-poc/kernelci-pipeline/src/fstests'
+VM_ROOT = '/home/kernelci/alex/fstests-poc/fstests/'
```

Comment:
I don't think we need these absolute paths (and we can't use it anyway as it's only valid on the `kernelci2` VM).  How about storing the JSON file in the kernel source tree like the KUnit template is doing?

Reply:
Sounds like a good idea